.gitignore

```diff
@@ -3,6 +3,7 @@
 
 # Eclipse, Netbeans and IntelliJ files
 /.*
+!.jenkins
```

Comment:
A very minor thing here, if this is a legitimate part of the project, I wouldn't hide it. Also I would use a generic name for the folder, like for example `ci`. Anyway is a good move to stop polluting the root folder of the project :)

Reply:
probably `.ci` is a more generic one and can be used for future purposes not related just with Jenkins, so @almope is probably right